### PR TITLE
Py3 conversion

### PR DESCRIPTION
**Pull request by edit4ever**
This is the initial Py3 conversion for LE Settings.
~~Bluetooth is broken due to dbus issue - but~~ everything ~~else~~ should work for testing.

### CODE REVIEW

File: src/resources/lib/modules/connman.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import xbmcgui
 import threading
 import oeWindows
-import ConfigParser
+import configparser
```
Comment (MilhouseVH) [inline src/resources/lib/modules/connman.py]: Where is `configparser` being used? Looks like we can do this `import`?

Comment (MilhouseVH) [reply src/resources/lib/modules/connman.py]: s/do/drop/

Comment (edit4ever) [reply src/resources/lib/modules/connman.py]: I found the reference all the way back OpenELEC 5 - under the load_vpn_config function.

I don't believe this module is needed - I would say we can test dropping it.

Comment (MilhouseVH) [reply src/resources/lib/modules/connman.py]: Yeah, should have been removed as part of this commit:

https://github.com/OpenELEC/service.openelec.settings/commit/5a97990b6fa87b8a09ac7a4ce5d4ec62778340b5

**Pull request merged**